app/controllers/kaui/accounts_controller.rb

```diff
@@ -38,8 +38,18 @@ def pagination
     end
 
     formatter = lambda do |account|
+      account_close_link = ''
```

Comment:
I think it would be nice to also have the ❌ on the main account screen (`/accounts/<ACCOUNT_ID>`), maybe next to the `Edit` link?